main/data/default_encryption_dialog.ui

```diff
@@ -0,0 +1,78 @@
+    <property name="default_height">260</property>
+    <child type="titlebar">
+      <placeholder/>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkBox">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="orientation">vertical</property>
+            <child>
+              <object class="GtkLabel" id="default_encryption_warning_label">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">No default end-to-end encryption method has been previously selected for this conversation. XMPP defines some XEP for end-to-end encryption that Dino supports.
+
+It is strongly recommended that one of the following end-to-end encryption methods below is selected before sending a message.
+
+TAKE INTO ACCOUNT UNENCRYPTED CONVERSATIONS COULD BE READ BY THIRD PARTIES.
+
+These settings can be later changed from the Settings menu.</property>
+                <property name="wrap">True</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="omemo">
+                <property name="label" translatable="yes">OMEMO (XEP-0384)</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
+                <property name="active">False</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="openpgp">
+                <property name="label" translatable="yes">OpenPGP (XEP-0027)</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
+                <property name="active">False</property>
+                <property name="group">omemo</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="none">
+                <property name="label" translatable="yes">Unencrypted (not recommended)</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
+                <property name="active">False</property>
+                <property name="group">omemo</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkButton" id="accept_button">
+                <property name="label" translatable="yes">Accept</property>
```

Comment:
This verb feels a bit out of place here. Normally, "accept" is used in dialogs where you just acknowledge an information or you choose between accepting and rejecting/cancelling.

You rather make a configuration and tell the app to apply it.

```suggestion
                <property name="label" translatable="yes">Apply</property>
```

Reply:
I agree with your suggestion. Thanks!